sdk/storage/azure-storage-files-datalake/src/datalake_file_client.cpp

```diff
@@ -52,6 +52,7 @@ namespace Azure { namespace Storage { namespace Files { namespace DataLake {
       ret.ContentEncoding = std::move(headers.ContentEncoding);
       ret.ContentLanguage = std::move(headers.ContentLanguage);
       ret.ContentType = std::move(headers.ContentType);
+      ret.ContentHash = std::move(headers.ContentHash);
```

Comment:
Was this a bug since content hash was missing? Is there a unit test for this. Please avoid such changes in a large PR like this as it hides semantic differences that need to be reviewed carefully and tested.

Reply:
@katmsft Can you answer this question?